dropshot/data/dont_use_this_key.rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Same comment here -- can we generate this as needed?  And if not, can we put a comment here explaining how it's used and how to generate it?